src/extension.ts

```diff
@@ -171,10 +169,13 @@ async function sleep(ms: number) {
 }
 
 async function finalizeUp(namespace: string, name: string) {
-    reporter.track(events.upReady);               
-    
-    try{
-        await vscode.commands.executeCommand("opensshremotes.openEmptyWindow", {hostName: name});
+    reporter.track(events.upReady);
+
+    try {
+        const remote = `${name}.okteto`;
```

Comment:
@rberrelleza I assume here that the okteto cli adds a host with this format to `.ssh/config`. Is this correct?
Also I set `/okteto` as the default folder. Right now if this folder doesn't exist the connection will fail and the VSCode UI doesn't allow to select a different folder...

Reply:
that's ok. we'll need to update it to get the working directory from okteto.yaml and if it doesnt exist, then use /okteto.

Reply:
> I assume here that the okteto cli adds a host with this format to .ssh/config. Is this always correct?